feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/validators/current/CurrentValidatorsViewModel.kt

```diff
@@ -69,7 +72,7 @@ class CurrentValidatorsViewModel(
         .asLiveData()
 
     val oversubscribedValidatorsFlow = groupedCurrentValidatorsFlow.map {
-        it[NominatedValidator.Status.Active]?.get(0)?.validator?.electedInfo?.isOversubscribed() ?: false
+        it[NominatedValidator.Status.Active]?.get(0)?.validator?.electedInfo?.isOversubscribed ?: false
```

Comment:
What if the second validator is oversubscribed?

`it[NominatedValidator.Status.Active]?.any{ condition }` should be correct

Reply:
Done